### ISSUE TITLE
GravitySnapHelper ignores RecyclerView padding in 2.0

### ISSUE THREAD

**Issue by CodeBreak524** [rubensousa/RecyclerViewSnap#58]
In version 1.5 GravitySnapHelper respected paddings of RecyclerView, which it was attached to. For example, using helper Gravity.START to recycler with paddingLeft > 0 was correctly handled, i.e. this padding remained while snapping to next elements. 
But in 2.0 this padding is ignored while snapping and this leads to layout break.

Example of used RecyclerView:
```
GravitySnapHelper(Gravity.START).attachToRecyclerView(recyclerView)

<androidx.recyclerview.widget.RecyclerView xmlns:android="http://schemas.android.com/apk/res/android"
    android:id="@+id/recyclerView"
    android:layout_width="match_parent"
    android:layout_height="100dp"
    android:layout_marginLeft="0dp"
    android:layout_marginRight="0dp"
    android:clipToPadding="false"
    android:nestedScrollingEnabled="false"
    android:paddingLeft="5dp"
    android:paddingRight="5dp" />
```

Layout of item used seems to be insignificant for this.

**Comment by rubensousa** [comment]
I just checked this using the sample. There's a list titled "Snap start with padding" and it seems to work fine.

Can you provide a screenshot of the behavior you're looking for VS what it's like at the moment?

**Comment by VinhTranVN** [comment]
Hello @rubensousa, I have the same issue above on 2.1.0, it worked OK on 1.5, you can check the attached images, I hope they can provide more info.

version 1.5
<img width="480" alt="1 5" src="https://user-images.githubusercontent.com/6914963/63826627-3dbd5980-c98a-11e9-8358-7f3916a277a4.png">

version 2.1.0
<img width="478" alt="2 1 0" src="https://user-images.githubusercontent.com/6914963/63826643-4d3ca280-c98a-11e9-80fd-a95e1196c8c6.png">

**Comment by rubensousa** [comment]
@VinhTranVN thank you for the example :)

I'll check this once I have more time.

**Comment by rubensousa** [comment]
Released in 2.2.0. Use this: https://github.com/rubensousa/GravitySnapHelper/blob/2.2.0/gravitysnaphelper/src/main/java/com/github/rubensousa/gravitysnaphelper/GravitySnapHelper.java#L324

By default this is false, so you need to call that and pass true
